sample-on-prem-ha-yamls/onprem-ha-blobs-nfs-persistent-volume.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright Sonatype, Inc., 2022, All Rights Reserved.
+#
+# Note: This YAML file is just an example and cannot be used as-is.
+# You must fill it out with the appropriate information for your deployment to be able to use it.
+
+kind: PersistentVolume
```

Comment:
The NFS mount is really optional, users may be using an appliance with an S3-style API.